tdrs-frontend/.pa11yci.json

```diff
@@ -22,25 +22,40 @@
       "screenCapture": "pa11y-screenshots/homepage.png"
     },
     {
+      "actions":[
```

Comment:
These actions should let us know when pa11y can't get past the welcome screen with out an error in the future, and prevent us from committing work that breaks pa11y.

Reply:
Just for DRY purposes, is it possible to have all actions under a single "wait for element" block?

Reply:
Not as a config file, we would need to use it through js, or at least involve some kind of code.